Formula/opentsdb.rb

```diff
@@ -0,0 +1,121 @@
+require "socket"
+
+class Opentsdb < Formula
+  desc "Scalable, distributed Time Series Database."
+  homepage "http://opentsdb.net/"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.2.0/opentsdb-2.2.0.tar.gz"
+  sha256 "5689d4d83ee21f1ce5892d064d6738bfa9fdef99f106f45d5c38eefb9476dfb5"
+
+  option "without-lzo", "Don't use LZO Compression"
+
+  if build.without? "lzo"
+    depends_on "hbase"
+  else
+    depends_on "hbase" => "with-lzo"
```

Comment:
I just wanted to verify the correct way to specify this. If you installed hbase with lzo but (for some reason) don't want lzo for opentsdb you shouldn't be forced to reinstall hbase without lzo, that's silly. But using lzo with opentsdb does require that you installed hbase with lzo.

My reasoning is a bit convoluted, sorry.